### DISCUSSION TITLE
Requesting   FIND function in Editor

### DISCUSSION THREAD

**Discussion by OrangeBox72** [paulscottrobson/neo6502-firmware#514] [Ideas]
It would be nice to have a FIND function in the built-in Editor.
ie.   CTRL-F  'SEARCH-TEXT'

**Comment by paulscottrobson** [comment]
It's possible, but non trivial because of the way the editor operates, and might be quite slow, because you'd have to detokenise every line to do it ; there's never more than one line in actual text state at once. I'll give it some thought
